### ISSUE TITLE
How to use Blink.Window()

### ISSUE THREAD

**Issue by barygold** [JuliaGizmos/Blink.jl#316]
There are many parameters that can be set when creating a window.
Please provide an example where width=200 and height=100 when using Blink.Window().

**Comment by ujimushi** [comment]
```julia
using Blink
win = Window(Dict(:width=>200, :height=>100))
```
It's OK? (Sorry. I'm not good at English.)

**Comment by Vectornaut** [comment]
Yes, @ujimushi, I think that's what the original poster was asking for. This should be included in the documentation!
